style.css

```diff
@@ -1,3 +1,3 @@
 h1 {
-    color: chartreuse;
-}
\ No newline at end of file
+  color: chartreuse;
+}
```

Comment:
- [ ] Good job for fixing the stylint error requested by previous reviewer.
- [ optional ] Please try to use class for the header as suggested by the  previous reviewer.